crates/lsp/src/snapshots/lsp__handlers_format__tests__format_range_logical_lines-4.snap

```diff
@@ -4,5 +4,5 @@ expression: output
 ---
 1+1
 {
-  2 + 2
+	2 + 2
```

Comment:
The LSP was hardcoded to `Space` before but we are going to get the default settings now, which are currently `Tab`